router/internal/unsafebytes/unsafebytes.go

```diff
@@ -1,52 +1,18 @@
 package unsafebytes
 
 import (
-	"reflect"
-	"strconv"
 	"unsafe"
 )
 
-func BytesToInt64(byteSlice []byte) int64 {
-	out, _ := strconv.ParseInt(*(*string)(unsafe.Pointer(&byteSlice)), 10, 64)
-	return out
-}
-
-func BytesToFloat32(byteSlice []byte) float32 {
-	out, _ := strconv.ParseFloat(*(*string)(unsafe.Pointer(&byteSlice)), 64)
-	return float32(out)
-}
-
+// BytesToString converts a byte slice to a string without a memory allocation.
```

Comment:
Available since go 1.20 https://blog.devops.dev/fast-string-to-byte-and-byte-to-string-conversion-in-go-1-20-85bdb859ee67

Reply:
To fix vet issues